examples.py

```diff
@@ -23,10 +24,16 @@ def multiple():
     print('\t'.join(emails))
     print('\t'.join(i and 'valid' or 'invalid' for i in verify_email(emails)))
 
+async def main():
+    print('\t'.join(emails))
+    print('\t'.join(i and 'valid' or 'invalid' for i in await verify_email_async(emails)))
+
 
 print('### single')
 for email in emails:
     time_it(single, email)
 print('### multiple')
 time_it(multiple)
 print('note that network condition may have impact on test results')
+print("asyncio test")
+asyncio.get_event_loop().run_until_complete(main())
```

Comment:
This is the reason, I was not ready to convert the method `verify_email` into a coroutine, as it's not expected from a beginner to know about coroutine.
But thanks for adding that!